### PR TITLE
add build in traffic light recognition&preprocessor

### PR DESCRIPTION
**Pull request by xiaoMrzhang**

### CODE REVIEW

**Comment by storypku** [comment]
It seems that its your first few PRs. Welcome on board!

First, please fix those lint errors by running `scripts/clang_format.sh path/to/A.cc path/to/B.h`.

Second, your commit msg (when running `git commit`) should be like  "module-name:  blahblahblah" ( `Perception: add BUILD files for traffic light` in your case, for example) to differentiate it from commits for other modules.

Please ask other members for algorithmic review.

File: modules/perception/camera/lib/traffic_light/detector/recognition/BUILD [hunk 1]
```diff
@@ -12,10 +12,10 @@ cc_library(
         "//modules/perception/camera/common",
         "//modules/perception/camera/lib/interface",
         "//modules/perception/camera/lib/traffic_light/detector/recognition/proto:recognition_cc_proto",
-        "//modules/perception/inference:inference_factory",
-        "//modules/perception/inference:inference_lib",
-        "//modules/perception/inference/utils:inference_resize_lib",
-        "//modules/perception/inference/utils:inference_util_lib",
+        # "//modules/perception/inference:inference_factory",
```
Comment (storypku) [inline modules/perception/camera/lib/traffic_light/detector/recognition/BUILD:15]: Just remove them rather than comment out.

**Review by storypku — CHANGES_REQUESTED** [review]

File: modules/perception/camera/lib/traffic_light/detector/recognition/BUILD (hunk 1 above)
Comment (jinghaomiao) [reply modules/perception/camera/lib/traffic_light/detector/recognition/BUILD:15]: Agree with @storypku . Leave TODO if you need to revisit it at a later time, or simply remove if if it's no longer needed.

**Comment by lx18233184051** [comment]
traffic_light/detector/recognition:classify cpplint
traffic_light/preprocessor:multi_camera_projection cpplint
traffic_light/preprocessor:tl_preprocessor cpplint
the three files have some format problems need to be modified.

**Pull request closed without merging**